### DISCUSSION TITLE
VideoDownloadHelper fails when trying to download from OnlyFans

### DISCUSSION THREAD

**Discussion by Jules337** [aclap-dev/video-downloadhelper#1170] [Help and Q&A]
### Your question

I think before the newest version of this I didn't seem to have any issues but now it fails to download and also doesn't pickup multiple videos from a post, can anyone advise? thanks

### Addon & CoApp version and details

version: 9.1.0.42
target: mozilla
channel: stable
lang: en-GB
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:124.0) Gecko/20100101 Firefox/124.0
{
    "used_history_button": true,
    "view_options": {
        "all_tabs": true,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "open_count_store": 15,
    "record_download_history": true
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

There have been other reports on here like this.  You should search this forum, find them, & read them.
